src/pages.c

```diff
@@ -312,6 +312,29 @@ pages_nohuge(void *addr, size_t size) {
 #endif
 }
 
+bool
+pages_dontdump(void *addr, size_t size) {
+	assert(PAGE_ADDR2BASE(addr) == addr);
+	assert(PAGE_CEILING(size) == size);
+#ifdef JEMALLOC_DONTDUMP
```

Comment:
I might be missing something. Where is this defined?

Reply:
Woops, good catch. Should be JEMALLOC_MADVISE_DONTDUMP, from jemalloc_internal_defs.h.

Reply:
It looks this one should be changed as well.

Reply:
Ugh. Fixed that one too. Sorry.